app/upgrades/v2/upgrades.go

```diff
@@ -67,7 +70,7 @@ func CreateUpgrade(includeAsyncICQ bool) upgrades.Upgrade {
 		CreateUpgradeHandler: CreateUpgradeHandler,
 		StoreUpgrades: store.StoreUpgrades{
 			Added:   addedStoreUpgrades,
-			Deleted: []string{ibcfeetypes.StoreKey},
+			Deleted: []string{ibcfeetypes.StoreKey, CrisisStoreName},
```

Comment:
Do we need to do the same thing we did with `addedStoreUpgrades`? Given that the testnet upgrade did not include the Crisis module being removed.